starport/services/network/networktypes/genesisinformation.go

```diff
@@ -77,59 +77,136 @@ func NewGenesisInformation(
 	vestingAccs []VestingAccount,
 	genVals []GenesisValidator,
 ) (gi GenesisInformation) {
-
-	// convert account arrays into maps
-	gi.GenesisAccounts = make(map[string]GenesisAccount)
-	for _, genAcc := range genAccs {
-		gi.GenesisAccounts[genAcc.Address] = genAcc
-	}
-	gi.VestingAccounts = make(map[string]VestingAccount)
-	for _, vestingAcc := range vestingAccs {
-		gi.VestingAccounts[vestingAcc.Address] = vestingAcc
-	}
-	gi.GenesisValidators = make(map[string]GenesisValidator)
-	for _, genVal := range genVals {
-		gi.GenesisValidators[genVal.Address] = genVal
-	}
+	gi.GenesisAccounts = make([]GenesisAccount, len(genAccs))
+	gi.VestingAccounts = make([]VestingAccount, len(vestingAccs))
+	gi.GenesisValidators = make([]GenesisValidator, len(genVals))
+	copy(gi.GenesisAccounts, genAccs)
+	copy(gi.VestingAccounts, vestingAccs)
+	copy(gi.GenesisValidators, genVals)
 	return gi
 }
 
 // GetGenesisAccounts converts into array and returns genesis accounts
 func (gi GenesisInformation) GetGenesisAccounts() (accs []GenesisAccount) {
-	for _, genAcc := range gi.GenesisAccounts {
-		accs = append(accs, genAcc)
-	}
+	accs = make([]GenesisAccount, len(gi.GenesisAccounts))
```

Comment:
Can't see why do we need these getters, we can directly access to the genesis accounts through `GenesisInformation.GenesisAccounts`.

Can we remove this and alike funcs?